app/components/TransactionModal.tsx

```diff
@@ -38,9 +38,10 @@ export const TransactionModal = ({
     <Dialog open={isModalOpen(ModalId.WAITING_TRANSACTION)} onOpenChange={close}>
       <DialogContent>
         <DialogHeader>
+          {/* This looks akward */}
           <VisuallyHidden asChild>
             <>
-              <DialogTitle>Transaction Modal</DialogTitle>
+              <DialogTitle></DialogTitle>
```

Comment:
why?

Reply:
I think we need this to describe the aria label

Reply:
1. This is not being properly hidden, show "Transaction Modal" text appears.
2. If you hide the tittle, the modal layout breaks (close button goes to the left). Maybe there is another solution or we just need to improve the lib component.

Reply:
that's odd, cause the whole purpose we add it was for having hidden element to do descriptions, maybe it's because of `asChild` props?  (if you could try)

We should fix this (in another PR) not sure if we should change it in this PR tho.